app/assets/javascripts/discourse/tests/integration/components/color-input-test.gjs

```diff
@@ -28,4 +28,49 @@ module("Integration | Component | ColorInput", function (hooks) {
     await triggerEvent(".hex-input", "blur");
     assert.strictEqual(result, "052e3d", "with 6 digit hex");
   });
+
+  module("with fallback hex value provided", function (innerHooks) {
```

Comment:
```suggestion
  module("with fallback hex value provided", function (hooks) {
```

Seems like we only use `innerHooks` in a few places, `hooks` is much more common